tools/hlt_client/hlt_client/client.py

```diff
@@ -184,6 +184,9 @@ def _parse_arguments():
                                     help='Number of replays to fetch')
     replay_user_parser.add_argument('-d', '--destination', dest='destination', action='store', type=str, required=True,
                                     help="In which folder to store all resulting replay files.")
+    replay_user_parser.add_argument('-t', '--date', action='store', type=str, dest='date', required=True,
```

Comment:
This should not be a required argument.

Reply:
Okay, I will change it to 'default=None' and remove 'required=True'.